pages/dkp/kaptain/1.0.1-0.6.0/install/konvoy/index.md

```diff
@@ -8,18 +8,33 @@ beta: false
 enterprise: false
 ---
 
-* After the Konvoy cluster has been deployed successfully (including Istio and Knative), install KUDO:
+## Install
+
+* Add Kaptain addon repository in your Konvoy `cluster.yaml` to install Kaptain dependencies,
+then follow the Konvoy documentation to deploy the addons:
+  ```yaml
+      - configRepository: https://github.com/mesosphere/kubeaddons-kaptain
+        configVersion: stable-1.17-0.5.0
+        addonsList:
+          - name: knative
+            enabled: true
+          - name: kubeflow-nfs
+            enabled: true
+  ```
+* Please note that KUDO parameters are not supported in this installation option due to Konvoy API limitation.
```

Comment:
I think this is a redundant piece of information. The sentence describes the situation when Kaptain is installed as an addon (this is where the limitation applies). However, in this and some other guides, the instructions describe Kaptain installation from a KUDO package tarball which doesn't have this problem. I'd suggest removing this list item to avoid confusion.